src/sentry/workflow_engine/endpoints/serializers.py

```diff
@@ -40,6 +40,66 @@ def serialize(self, obj: Action, *args, **kwargs) -> ActionSerializerResponse:
         }
 
 
+class SentryAppContext(TypedDict):
+    id: str
+    name: str
+    installationId: str
+    status: int
+    settings: NotRequired[dict[str, Any]]
+
+
+class ActionHandlerSerializerResponse(TypedDict):
+    type: str
+    handlerGroup: str
+    configSchema: dict
+    dataSchema: dict
+    sentryApp: NotRequired[SentryAppContext]
+    integrations: NotRequired[list]
+
+
+@register(ActionHandler)
```

Comment:
👍 -- groks a lot easier when looking at code that this isn't the same as an Action -- ty!